### ISSUE TITLE
Raspbian Bullseye: "Can't open display"?

### ISSUE THREAD

**Issue by rsend** [AndrewFromMelbourne/dmxwebcam#12]
Hi, I tried using this project on Bullseye the other day, but I haven't had any luck actually running it. The only thing that happens, no matter what options I start the program with, is that I get the error "Can't open display".

I tried on a Lite installation with no X or desktop, as well as a full LDXE environment - same result.

Has something changed lately on the OS side that would cause this issue?

**Comment by AndrewFromMelbourne** [comment]
Yes, the DispmanX  interface is no longer supported in Raspberry Pi OS.
